src/ledger/LedgerManagerImpl.cpp

```diff
@@ -408,6 +433,12 @@ LedgerManagerImpl::getLastClosedLedgerHeader() const
 HistoryArchiveState
 LedgerManagerImpl::getLastClosedLedgerHAS()
 {
+    if (rebuildingInMemoryState())
```

Comment:
did you need to special case this here? We could not initialize the instance with `has = {}` in this case instead?